spec/checkPullRequestReviewSpec.js

```diff
@@ -1000,7 +1023,333 @@ describe('Pull Request Review Module', () => {
     });
 
     afterAll(() => {
-      payloadData.payload.pull_request.state = initalState;
+      reviewPayloadData.payload.pull_request.state = initalState;
+    });
+  });
+
+  describe('Pull request author comments on PR', () => {
+    beforeEach(() => {
+      github.pulls = {
+        get: jasmine.createSpy('get').and.resolveTo({
+          data: commentPayloadData.payload.issue,
+        }),
+      };
     });
+
+    describe(
+      'when author asks reviewer to ptal and does not assign them and author is assigned', () => {
```

Comment:
Break line - please check the complete file.

Reply:
Done.